aiidalab_widgets_base/viewers.py

```diff
@@ -759,10 +796,10 @@ def _observe_structure(self, change):
         # Remove the current structure(s) from the viewer.
         if change["new"] is not None:
             self.set_trait("displayed_structure", change["new"].repeat(self.supercell))
-            self.set_trait("cell", change["new"].cell)
+            self.set_trait("ase_structure", change["new"])
```

Comment:
I don't think this works in general case because the `structure` trait can be AiiDA node (StructureData or CifData). You would need to move this to the `displayed_structure` observer, but as mentioned in the other comment, I think it is easier to just move the `displayed_structure` trait to the parent class.